lib/support/cli.js

```diff
@@ -447,6 +447,11 @@ module.exports.parseCommandLine = function parseCommandLine() {
     .option('resultBaseURL', {
       describe: 'The base URL to the server serving the HTML result. In the format of https://result.sitespeed.io'
     })
+    .option('gzipHAR', {
+      describe: 'Compress the HAR files with GZIP.',
+      default: false,
```

Comment:
We will change the current behavior if we have it on by default. It adds one extra step if you download and wanna drag/drop it to another tool, like the maybe coming "compare 2 HARs" :)